src/components/sbb-form-field/sbb-form-field.scss

```diff
@@ -119,6 +125,7 @@
 ::slotted(*[slot='suffix']) {
   min-width: var(--sbb-icon-svg-width);
   display: flex;
+  color: var(--sbb-form-field-prefix-color);
 }
```

Comment:
It's strange for me that a "prefix" color is applied to the "suffix" and not anymore to the "prefix"?

Reply:
Also, comparing to figma, this seems to introduce a bug.